### PR TITLE
Profile highlight & Profile key metrics UI changes

### PR DESCRIPTION
**Pull request by emre2038**
## Description
Making UI changes in profile highlight & profile key metrics 

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-348

## How to test it locally
* use https://wazimap-pr-417.herokuapp.com/admin/ to test this PR
* go to profile highlights page
* confirm that there are no section headings other than `Profile fields` and `Change reason`
* confirm that the fields are ordered correctly
   - Profile
   - Subcategory
   - Label
   - Variable stuff
 * confirm that helptext is added for Profile selection
 * confirm that once it is saved, Profile is readonly
 * repeat these for profile key metrics page

image

image

### Updated
* `wazimap_ng/profile/models.py` to add help texts
* `wazimap_ng/profile/admin/admins/profile_highlight_admin.py` & `wazimap_ng/profile/admin/admins/profile_key_metrics_admin.py` to make UI changes

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out

### Commits

- [x]  commits are clean
- [x]  commit messages are clean

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers
- [ ]  black was run locally (as part of the pre-commit hook)

### Testing

- [x]  ✅ added (appropriate) unit tests
- [x]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

File: wazimap_ng/profile/admin/admins/profile_key_metrics_admin.py [hunk 1]
```diff
@@ -37,16 +42,18 @@ class ProfileKeyMetricsAdmin(SortableAdminMixin, BaseAdminModel, HistoryAdmin):
...
     help_texts = ["denominator", ]
 
-    fields = ["profile", "variable", "subindicator", "subcategory", "denominator", "label"]
-    search_fields = ("label", )
+    search_fields = ("label",)
 
+    def get_readonly_fields(self, request, obj=None):
+        if obj:  # editing an existing object
+            return ("profile",) + self.readonly_fields
+        return self.readonly_fields
```
Comment (michaelglenister) [inline wazimap_ng/profile/admin/admins/profile_key_metrics_admin.py:50]: The codeclimate warning about repeating code might actually be correct here. But it can stay like this for now

**Review by michaelglenister — APPROVED** [review]
LGTM, thanks
